### PR TITLE
Marker navigation

### PR DESCRIPTION
**Pull request by williamsyang-work**
Implements functionality to navigate through markers by using "," and ".".

Fixes the TODO's listed in PR #237.  Uses class inheritance to remove duplicate code.  Uses different algorithm for computing next/previous state, since marker states don't always come in a time-sorted array.  

Signed-off-by: Will Yang <william.yang@ericsson.com>

### CODE REVIEW

**Review by bhufmann — APPROVED** [review]
Looks good to me. Thanks for this feature!

File: timeline-chart/src/layer/time-graph-markers-chart-cursors.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+            if (isValid && closestState) {
+                this.unitController.selectionRange = closestState.range;
+                this.maybeCenterCursor();
+                this.update();
+            }
+
+        }
+    }
+
+}
```
Comment (PatrickTasse) [inline timeline-chart/src/layer/time-graph-markers-chart-cursors.ts:67]: It's asking for a newline at EOF

**Review by PatrickTasse — CHANGES_REQUESTED** [review]
Please squash with previous commit in PR

**Comment by bhufmann** [comment]
> Please squash with previous commit in PR

Makes sense. My commit doesn't need to be merged.

**Comment by williamsyang-work** [comment]
New line added and commits squashed.

**Review by bhufmann — APPROVED** [review]
Still looks good to me.

**Pull request merged**